app/assets/javascripts/discourse/app/components/sidebar/common/categories-section.js

```diff
@@ -38,7 +38,7 @@ export default class SidebarCommonCategoriesSection extends Component {
       return this.categories;
     }
 
-    let categories = this.site.categories;
+    let categories = [...this.site.categories];
 
     if (!this.siteSettings.fixed_category_positions) {
       categories = categories.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
To make it clearer this modifies the array in-place 😃 

```suggestion
      categories.sort((a, b) => a.name.localeCompare(b.name));
```

Reply:
O this is a good point :+1: Have made the changes.